lib/salus/scanners/yarn_audit.rb

```diff
@@ -127,6 +133,91 @@ def handle_legacy_yarn_audit
       report_failure
     end
 
+    def auto_fix_vulns
+      @vulns_w_paths.each do |vuln|
+        path = vuln["Path"]
+        package = vuln["Package"]
+        # vuln["Path"] for some indirect dependency "foo"
+        # looks like "foo > bar > baz".
+        # vuln["Path"] for some direct dependency "foo"
+        # looks like "foo"
+        is_direct_dep = path == package
+
+        # TODO: Fix indirect dependencies as well
+        fix_direct_dependency(vuln) if is_direct_dep
+      end
+    end
+
+    def fix_direct_dependency(vuln)
+      package = vuln["Package"]
+      patched_version_range = vuln["Patched in"]
```

Comment:
Could `vuln["Patched in"]` be nil or empty string?

Reply:
It could definitely be the latter.